hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/UtilVerbs.java

```diff
@@ -1082,21 +1082,26 @@ private static void reduceFeeComponentsFor(
     }
 
     public static HapiSpecOperation uploadDefaultFeeSchedules(String payer) {
+        return uploadCustomFeeSchedules(payer, "FeeSchedule.json");
+    }
+
+    public static HapiSpecOperation uploadCustomFeeSchedules(String payer, String feeSchedule) {
         return withOpContext((spec, opLog) -> {
-            allRunFor(spec, updateLargeFile(payer, FEE_SCHEDULE, defaultFeeSchedules()));
+            allRunFor(spec, updateLargeFile(payer, FEE_SCHEDULE, customFeeSchedules(feeSchedule)));
             if (!spec.tryReinitializingFees()) {
                 throw new IllegalStateException("New fee schedules won't be available, dying!");
             }
         });
     }
 
-    private static ByteString defaultFeeSchedules() {
+    private static ByteString customFeeSchedules(String feeSchedules) {
         SysFileSerde<String> serde = new FeesJsonToGrpcBytes();
         var baos = new ByteArrayOutputStream();
         try {
-            var schedulesIn = HapiFileCreate.class.getClassLoader().getResourceAsStream("FeeSchedule.json");
+            var schedulesIn = HapiFileCreate.class.getClassLoader().getResourceAsStream(feeSchedules);
             if (schedulesIn == null) {
-                throw new IllegalStateException("No FeeSchedule.json resource available!");
+                throw new IllegalStateException(
+                        String.format("No feeSchedule.json resource available for: %s", feeSchedules));
```

Comment:
Does the file name start with the lowercase letter?

Reply:
Yes, the original file in genesis, testnet and mainnet is named "feeSchedule.json".

Reply:
Nit: I believe concatenating the Strings is faster than using `String.format()` in this particular case.

Reply:
In our case, it works faster with `String. format()`. I tested with modified `customFeeSchedules()`. The changes are:  ```
long startTime, endTime;

            // Benchmark concatenation
            startTime = System.nanoTime();
            if (true) { // Replace with actual condition
                String info = "No feeSchedule.json resource available for: " + feeSchedules;
                System.out.println(info);
                endTime = System.nanoTime();
                System.out.println("Concatenation: " + (endTime - startTime) + " ns");
            } 
```. 
When I run `createHollowAccountWithMultipleSendersAndCompleteIt()` the result are:

- Concatenation: 131542 ns
- Concatenation: 146166 ns
- Concatenation: 94000 ns
- Concatenation: 107334 ns
- Concatenation: 136042 ns
 ~ 123 017 ns
 
- String.format(): 92292 ns
- String.format(): 69333 ns
- String.format(): 81250 ns
- String.format(): 72958 ns
- String.format(): 79292 ns
~ 79 025 ns

Reply:
Interesting result. I would have never expected this. I stand corrected. 🙂

We should not run microbenchmarks like this, though. The outcome can be flawed in too many ways. Instead, we should always use JMH.

Reply:
I conducted tests with JMH and found that string concatenation is faster with minimal values than string format.